section3/reflection.md

```diff
@@ -1,17 +1,21 @@
 ## Section 3 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
-
+`It generally felt the same. However, see question 2.`
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
-
+`I generally found myself wishing there was more time. Taking a break right at 25 minutes felt like I was relinquishing any mojo I might have had going. Also, I drink a *lot* of water, so taking regular breaks is generally good practice for me.``
 3. What is a Hash, and how is it different from an Array?
-
+`A hash is a string that is passed. It is cardinal, meaning it has no order. An array, however, is ordinal, and thus has an order.`
 4. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
 
-5. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
+`pet_store = {fish: 40, gerbils: 20, snakes: 5, hamsters: 15}`
 
+5. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
+`puts states.IA`
 6. With the same hash above, how would we get all the keys?  How about all the values?
-
+`puts pet_store.keys`
+`puts pet_store.values`
 7. What is another example of when we might use a hash?  In your example, why is a hash better than an array?
-
-8. What questions do you still have about hashes?
\ No newline at end of file
+`When we have values other than numbers. This is better, because an array only allows for numbers.`
+8. What questions do you still have about hashes?
+`I'm still trying to feel my way around all the methods.`
```

Comment:
Totally - you'll have SO much time in Mod 1 to get practice with these. We just scratched the surface - based on your work you are in a great place with them.